open_lm/model.py

```diff
@@ -19,6 +20,15 @@
 from open_lm.positional_embedding.rotary import RotaryWithCast
 from open_lm.positional_embedding.llama_rotary import LLaMARotaryWithCast
 
+
+# from open_lm.moe.mixture_of_experts import MoE
+try:
+    from megablocks.layers.moe import MoE
+    from megablocks.layers.arguments import Arguments as MoEArgs
+except ImportError:
+    logging.warning(f"Megablocks not installed. To train MoE, install with pip install megablocks.")
```

Comment:
Should we have some assert here to make sure they're not using MoE?

Reply:
we can't check args during imports though, right?